src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const headElements = document
+  .querySelector('thead')
+  .querySelectorAll('th');
+
+for (const item of headElements) {
+  item.addEventListener('click', () => {
+    const table
+      = [...document.querySelector('tbody').querySelectorAll('tr')]
```

Comment:
I suggest to move it into variable
`const bodyRows = document.querySelector('tbody').querySelectorAll('tr')`